### PR TITLE
Add los_record endpoint

### PR DESCRIPTION
**Pull request by Fredar**

### CODE REVIEW

File: content/reference/endpoints/los_records.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  <li class="active"><a href="#public" role="tab" data-toggle="pill">public</a></li>
+</ul>
+<div class="tab-content" markdown="1">
+  <div class="tab-pane active" id="public" markdown="1">
+Name                 | Type    | Read/Write | Description
+---------------------|---------|------------|------------
+id                   | Integer | Read       | Los Record's id.
+---------------------|---------|------------|------------
+day                  | [Date](/reference/enums#formats) | Read       | Day for which Los Record is valid.
+kind                 | String  | Read       | Los Record's kind, either `initial` or `final`.
```
Comment (ZenCocoon) [inline content/reference/endpoints/los_records.md]: I think it would be best to extract in enum so we can better describe each kind.

Comment (Fredar) [inline content/reference/endpoints/los_records.md]: Definitions below are not accurate? 
image

Taken from `rates` endpoint.

Comment (ZenCocoon) [inline content/reference/endpoints/los_records.md]: I did not sport them before sorry. that should work fine.

Comment (Mandaryn) [inline content/reference/endpoints/los_records.md]: @ZenCocoon didn't you prepare some more detailed description with flo on how initial vs final works? Guess its a good place to describe it in details

**Comment by Azdaroth** [comment]
What's the status of this one? It's pretty old. @Fredar @ZenCocoon

**Comment by ZenCocoon** [comment]
I think the initial price definition is no longer right. It should be the price of the rental only, special offers included.
It then ignores fees and excluded taxes

**Comment by Mandaryn** [comment]
not sure what it should or should not include, that be a business problem.
But currently initial price is rental price without special offer, and final price is rental price with special offer, neither of them including any fees or taxes

**Comment by ZenCocoon** [comment]
Ok, that need to be changed then. We have to follow the same logic as the rentals search endpoint.

File: content/reference/endpoints/los_records.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+<div class="callout callout-info">
+  <h4>Definitions</h4>
+  <strong>Initial los record</strong>: Los Record computed before any discount.<br>
+  <strong>Final los record</strong>: Los Record after discounts have been applied.
+</div>
+
+## List los records
+
+List all los records for given account(s).
```
Comment (thilonel) [inline content/reference/endpoints/los_records.md]: Why the `(s)`?

Comment (Fredar) [inline content/reference/endpoints/los_records.md]: Its like that all over the doc, consistency

Comment (thilonel) [inline content/reference/endpoints/los_records.md]: Can you actually list for more accounts? o.O If not, then it's misleading.

**Comment by thilonel** [comment]
When you're using the API, it can be useful, if you'd like to query just that rental's LOS. Anyhow, what would we gain by removing?

**Comment by Mandaryn** [comment]
its not scoping to rental, but fetching a single los for a single day and type by id (which is never done in my apps, nor i can imagine anyone sane doing), we could remove some specs from core, some text here, generally the less the better, right?

**Comment by Fredar** [comment]
I think we should leave the show, so this works the same as other endpoints.
Still could be used for some troubleshooting?

**Comment by Mandaryn** [comment]
troubleshooting? now you are both getting me convinced its completely useless :P

**Comment by ZenCocoon** [comment]
Actually I see that `rentals` don't have `links` to LOS records, which might be missing actually. However, when this is added, it means that some API users doing atomic fetch, like Rentals United, do need such endpoint.

**Comment by Mandaryn** [comment]
atomic fetch? can you give me a use case of anyone fetching los records one by one by id? they would need to fetch the entire collection first anyway, to get the ids...
Rentals link to los_records collection might be a good thing, but that is separate form the los show action, or am i missing something?

**Comment by ZenCocoon** [comment]
yes, links in rentals is different, but as soon as you expose record ID it's good to be able to fetch by a given ID.

Atomic fetch, I now it's very heavy and had this talk with them, however they do this everywhere and can't change like that, they also like it this way.
Example, they fetch our availability maps one by one using `show`, which is very bad performance wise to me but sadly the only way to have them connect to us today.

**Comment by Mandaryn** [comment]
they will make thousands of calls to fetch los record by record? then we should definately remove it so they don't do this crime to our servers... :trollface:

**Comment by ZenCocoon** [comment]
hahaha

**Comment by ZenCocoon** [comment]
I still think it's a necessary endpoint though

**Pull request merged**